pythonequipmentdrivers/oscilloscope/Lecroy_WR8xxx.py

```diff
@@ -755,6 +770,52 @@ def get_channel_label(self, channel: int) -> str:
 
         return ' '.join(response.strip().split()[1:])
 
+    def set_channel_label_position(self, channel: int, position=0) -> None:
+        """
+        set_channel_label_position(channel, position)
+
+        updates the text label position on a channel specified by "channel"
+        with the position given in "position".
```

Comment:
give more information about how the number "position" relates to the position of the label. Is it a number of divisions, pixels, etc...

Also, the default value isn't specified in the docstring anywhere

Reply:
Updated docstring, specified units (seconds) and changed position variable to float